CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 	Please see OpenShift release notes for official changes\n<!--Latest hash: 76cc46b739c27281db0f7b2a23eb091b2a70f698-->
 ## 4.13
 
+### Data Enhancement
+- [#723](https://github.com/openshift/insights-operator/pull/723) Obfuscate HTTP_PROXY and HTTPS_PROXY env variables on containers
```

Comment:
First time someone updated changelog with a new PR. :) This is not a new data enhancement, but rather bugfix or other (and yes it corresponds to the PR template selection)

Reply:
New data enhancement is e.g new gatherer for XYZ.

Reply:
Ok, updating it right now to BugFix. Thanks.